_variables.scss

```diff
@@ -61,3 +61,49 @@ $lineThick:       2px;
 // line color
 $lineColorLight:  $colorLightSecondary;
 $lineColorDark:   $colorMedium;
+
+
+
+
+/*----- styles for footer redesign ----*/
+$gridGutterWidth: 12px;
+
+$containerMinWidth: 600px;
+$containerMaxWidth: 1344px;
+
+$breakPointsLg: 1024px;
+
+$gridSmColumns: 8;
+$gridLgColumns: 12;
+
+// New color names
+
+$white:       #fff;
```

Comment:
why not use this? https://github.com/zalora/global-scss/blob/ee50c86b7aa94c14380f428d44e7a139f8fb3127/_variables.scss#L6

when doing the style guide, we wanted to follow the design pattern. Every Website have primary color and secondary color and so on. And the variable name should not reflect the actual colour.